### DISCUSSION TITLE
Request: add --curl option to grabber.ts

### DISCUSSION THREAD

**Discussion by PopeyeTheSai10r** [iptv-org/community#15] [Ideas]
> npm run grab -- [options]

Is the new way to run the grabber and [README.md](https://github.com/freearhey/epg-grabber#readme) has been updated to reflect that.

I think having the --curl option would be helpful, specially when it comes to troubleshooting sources that stopped working.

**Comment by freearhey** [comment]
Added in https://github.com/iptv-org/epg/pull/2814
